Tests/Twig/Extension/RequireJSExtensionTest.php

```diff
@@ -33,12 +30,15 @@ class RequireJSExtensionTest extends \PHPUnit_Framework_TestCase
 
     /**
      * {@inheritDoc}
+     * @test
```

Comment:
The `setUp` method is not a test, so it shouldn't have the `@test` annotation. By adding this annotation, PHPUnit will try to run the method as a test and look for assertions (which won't exist here, causing false negatives in the test result). Also the method doesn't need to change to public, it can stay protected